scylla/Cargo.toml

```diff
@@ -39,7 +39,7 @@ rand = "0.8.3"
 thiserror = "1.0"
 itertools = "0.11.0"
 tracing = "0.1.36"
-chrono = { version = "0.4.20", default-features = false, features = ["clock"] }
+chrono = { version = "0.4", default-features = false, features = ["clock"] }
```

Comment:
Did you check if the driver builds with `0.4.0`?
I see that it was introduced in this commit: https://github.com/scylladb/scylla-rust-driver/commit/10debf60725e8531b1e99904a013da9837570765
And that in `0.4.0` there are no `MIN / MAX` items in `NaiveData` which are used by us here: https://github.com/scylladb/scylla-rust-driver/blob/87908f3c73cdece258d28b5b2191a843e7da3b98/scylla/src/transport/cql_types_test.rs#L301
See docs for chrono: https://docs.rs/chrono/0.4.0/chrono/naive/struct.NaiveDate.html

So before dropping requirements `chrono` / `time` or other requirements version, please check if everything still compiles and passes with such versions. You can do that e.g. by using exact match in Cargo.toml, e.g. `chrono = { version = "=0.4.0", default-features = false, features = ["clock"] }`